src/app/containers/Include/index.jsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/no-danger */
+import React, { useEffect, useState } from 'react';
+import 'isomorphic-fetch';
+import { string } from 'prop-types';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import webLogger from '#lib/logger.web';
+import useToggle from '../Toggle/useToggle';
+
+const logger = webLogger();
+
+const IncludeContainer = ({ href }) => {
+  const [markup, setMarkup] = useState(null);
+  const { enabled } = useToggle('styIncludes');
+
+  useEffect(() => {
+    const fetchMarkup = async () => {
+      const res = await fetch(href);
```

Comment:
Let's stick to _either_ async/await _or_ Promises. E.G.:

```js
useEffect(() => {
  fetch(href)
    .text()
    .then(html => setMarkup(html))
    .catch(e => logger.error(`HTTP Error: "${e}"`));
}, [href]);
```

Reply:
I agree here with the sentiment to be consistent with using either async/await or promises here but I would personally always favour async/await due to the additional help the javascript interpreter gives in making sure errors don't go unhandled, I posted on another review a bit more of an example to explain this: https://github.com/bbc/simorgh/pull/5544#discussion_r380244682 

The code here _does_ correctly handle errors but using the async/await syntax would leave the code in a better place to accomodate any future changes; guarding against incomplete error handling that could be introduced later.